templates/migration/templates/migrationservice-configmap.yaml

```diff
@@ -18,6 +18,14 @@ data:
   USERS_PUBLIC_PLAN_TECHNICAL_PASSWORD: "{{ .Values.publicPlanTechnicalPassword }}"
   USERS_PUBLIC_PLAN_TECHNICAL_EMAIL: "{{ .Values.publicPlanTechnicalEmail }}"
   USERS_PUBLIC_PLAN_TECHNICAL_TENANTID: "{{ .Values.publicPlanTechnicalTenantId }}"
+
+  WEBLATE_CONFIG_SERVER_URL: "{{ .Values.weblateServerUrl }}"
```

Comment:
And for the weblate server url you could use a default too:
`{{ .Values.migration.weblate.server.url | default (printf "%s.%s" "weblate" .Release.Namespace) }}`
which will end in:
`weblate.develop` and `weblate.staging` etc.